app/views/layouts/application.html.haml

```diff
@@ -11,28 +11,35 @@
       = javascript_include_tag '//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv.min.js'
 
   %body
-    .navbar.navbar-default.navbar-static-top.navbar-inverse
+    .navbar.navbar-default.navbar-inverse.navbar-fixed-top
       .container
         %button.navbar-toggle(type="button" data-toggle="collapse" data-target=".navbar-responsive-collapse")
           %span.icon-bar
           %span.icon-bar
           %span.icon-bar
         = link_to root_path, class: 'navbar-brand' do
           = image_tag('logo.png', class: 'img-responsive', width:'80px')
-        = link_to 'Recipes', recipes_path, class: 'navbar-brand' if user_signed_in?
-        .navbar-collapse.collapse.navbar-responsive-collapse
-          %ul.nav.navbar-nav.navbar-right
-            - if user_signed_in?
-              %li.navbar-text Welcome #{current_user.given_name} #{current_user.last_name}
-              %li= link_to 'Edit Profile', edit_user_registration_path
-              %li= link_to 'Logout', destroy_user_session_path, method: :delete, data: { confirm: 'Are you sure?' }
-            - else
-              %li= link_to 'Login', new_user_session_path
-              %li= link_to 'Sign Up', new_user_registration_path
+        .collapse.navbar-collapse.navbar-responsive-collapse
+          - if user_signed_in?
+            %ul.nav.navbar-nav.navbar-left
+              %li= link_to 'My Recipes', recipes_path
+              %li= link_to 'All Recipes', recipes_path
+              %li= link_to 'New Recipe', new_recipe_path
+            %ul.nav.navbar-nav.navbar-right
+              - if user_signed_in?
+                %li.navbar-text Welcome #{current_user.given_name} #{current_user.last_name}
+                %li= link_to 'Edit Profile', edit_user_registration_path
+                %li= link_to 'Logout', destroy_user_session_path, method: :delete, data: { confirm: 'Are you sure?' }
+              - else
+                %li= link_to 'Login', new_user_session_path
+                %li= link_to 'Sign Up', new_user_registration_path
     .container
       .row
         .col-lg-12
+          %br
+          %br
           - flash.each do |key, value|
+            %br
```

Comment:
There are margins and paddings in CSS for this kind of things, `br` is meant to break text into a new line.